Framework/Core/src/DataProcessingDevice.cxx

```diff
@@ -636,6 +638,9 @@ static auto forwardInputs = [](ServiceRegistryRef registry, TimesliceSlot slot,
   std::vector<fair::mq::Parts> forwardedParts;
   forwardedParts.resize(proxy.getNumForwards());
   std::vector<ChannelIndex> cachedForwardingChoices{};
+  O2_SIGNPOST_ID_GENERATE(sid, forwarding);
+  O2_SIGNPOST_START(forwarding, sid, "forwardInputs", "Starting forwarding for slot %zu with oldestTimeslice %zu %{public}s%{public}s%{public}s",
```

Comment:
why does this say with oldestTimeslice? Is this the timeslice of the forwarded messages? Because this is what I'd need to know.

Reply:
Depends on how the forwarding happens, IIRC. If it's delayed, it's the oldestPossibleTimeslice at that moment.

Reply:
ok, could we then print instead (or in addition) the timeslice of the data that is forwarded?